search-app/src/cmr/search/site/static.clj

```diff
@@ -2,33 +2,64 @@
   "The functions of this namespace are specifically responsible for returning
   ready-to-serve pages."
   (:require
+   [cheshire.core :as json]
+   [clj-http.client :as client]
    [cmr.common-app.static :as static]
-   [cmr.search.site.data :as data])
+   [cmr.common.log :refer :all]
+   [cmr.search.site.data :as data]
+   [cmr.search.site.static.directory :as directory]
+   [cmr.search.site.static.site :as site]
+   [cmr.search.site.util :as util]
+   [cmr.transmit.config :as transmit])
   (:gen-class))
 
+;; Contextual data that is used for static content in the absense of a system
+;; context, e.g., when run from the command line.
+(defrecord StaticContext [
+  ;; the deployment env; one of :prod, :uat, :sit, or :local
+  deployment-target
+  ;; the application which is being run with the context, e.g., :search
+  cmr-application
+  ;; the context of execution, e.g., :cli, when run from a system shell
+  execution-context
+  ;; a place to store context-specific data
+  static-data])
+
 (defn generate-api-docs
```

Comment:
General comment: we should make functions only invoked within the namespace private to make it easier to read and maintain.

Reply:
I'm of two minds on this ... I see where you're coming from, but I also like to call functions like this from the REPL ...

I'll go ahead and make these private, though; I can just make them public (temporarily) in the future if I need to (though here, I wouldn't really need to since ( could just call `(-main :api)`, etc.).